pkg/serviceaccount/claims.go

```diff
@@ -49,7 +49,7 @@ type kubernetes struct {
 	Namespace string          `json:"namespace,omitempty"`
 	Svcacct   ref             `json:"serviceaccount,omitempty"`
 	Pod       *ref            `json:"pod,omitempty"`
-	Secret    *ref            `json:"secret,omitempty"`
+	Secret    *ref            `json:"secret,omitempty" datapolicy:"token"`
```

Comment:
This is an object reference, so it just has a name and uid of a secret. It's not sensitive. The return of this function is the actual token:

https://github.com/kubernetes/kubernetes/blob/1968e961653e10d121d6941fed2c9ca68d719554/pkg/serviceaccount/jwt.go#L213

Reply:
ok, lgtm. Closing as this is the only line changed in this PR.